internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -349,6 +350,59 @@ void testGetRealTimeTopicNameWithNonHybridVersion() {
     assertEquals("TestStore" + Version.REAL_TIME_TOPIC_SUFFIX, result);
   }
 
+  @Test
+  void testRealTimeTopicNameWithHybridConfig() {
+    // Mock the Store and HybridStoreConfig
+    Store store = mock(Store.class);
+    HybridStoreConfig hybridStoreConfig = mock(HybridStoreConfig.class);
+
+    // Define behavior
+    when(store.getHybridStoreConfig()).thenReturn(hybridStoreConfig);
+    when(store.getName()).thenReturn("test-store");
+    when(hybridStoreConfig.getRealTimeTopicName()).thenReturn("real-time-topic");
+
+    // Test
+    String result = Utils.getRealTimeTopicNameFromStoreConfig(store);
+
+    // Validate
+    assertEquals("real-time-topic", result);
```

Comment:
in asserts `expectation` comes as second arg and we should follow the convention otherwise it's confusing to test failures. 
```
assertEquals(String actual, String expected)
```

Let's fix all the places to follow this pattern